distribution/tools/entitlement-agent/build.gradle

```diff
@@ -51,7 +51,9 @@ tasks.named('jar').configure {
   manifest {
     attributes(
       'Premain-Class': 'org.elasticsearch.entitlement.agent.EntitlementAgent'
```

Comment:
Do we still need to specify Premain-Class even though we now load dynamically?

Reply:
Nope! I was keeping options open, but actually static loading won't even work anymore, so I've deleted it now.